torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -952,6 +953,15 @@ Tensor* TensorExprKernel::computeValue(const torch::jit::Value* v) {
       });
     } break;
 
+    case aten::isnan: {
+      return computeOneOperand("aten_relu", v, [](const ExprHandle& a) {
```

Comment:
Maybe want to name this `aten_isnan` instead of `aten_relu`, although tbh it's been a TODO for a long time to give these silly things names corresponding to their IR values instead of their aten_op.

Reply:
oops lol, you can see where i started the implementation from